### PR TITLE
Github Action for publish to RubyGems

### LINKED ISSUE #9: Automate gem publishment
https://andrewm.codes/blog/automating-ruby-gem-releases-with-github-actions/

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #9.

### CODE REVIEW

**Comment by ksnitsky** [comment]
Code in action from Github Actions -> RubyGems Action in Github repository navigation panel.

File: lib/work_calendar/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module WorkCalendar
-  VERSION = '1.0.3'
+  VERSION = '1.0.4'
```
Comment (sergey-tomashevsky) [inline lib/work_calendar/version.rb:4]: @ksnitsky Have you manually changed gem version here? Any update on auto-increment?

**Review by sergey-tomashevsky — COMMENTED** [review]
Still waiting on Oleg's response to API key question. Will get updated when I have something.

File: .github/workflows/publish_to_rubygems.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Ruby 2.6
```
Comment (sergey-tomashevsky) [inline .github/workflows/publish_to_rubygems.yml]: Speaking of, can we use later Ruby version here? 2.6 is rather outdated.

File: work_calendar.gemspec [hunk 3]
```diff
@@ -27,10 +27,10 @@ Gem::Specification.new do |spec|
...
-  spec.add_development_dependency 'minitest', '~> 5.0'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rubocop', '~>0.71'
-  spec.add_development_dependency 'rubocop-performance', '~>1.3.0'
+  spec.add_development_dependency 'activesupport'
+  spec.add_development_dependency 'bundler'
+  spec.add_development_dependency 'minitest'
+  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'rubocop-performance'
```
Comment (ksnitsky) [inline work_calendar.gemspec]: Perhaps I should set fixed versions after the upgrade, as in the `.lock` file?

Comment (sergey-tomashevsky) [reply work_calendar.gemspec]: Does it matter though? We can keep versions unspecified so those will get updated during later changes.

**Review by sergey-tomashevsky — APPROVED** [review]

**Pull request merged**